.gitignore

```diff
@@ -12,6 +12,7 @@ src/backend/node_modules
 src/backend/.nyc_output
 src/backend/.env
 src/backend/dist
+src/backend/config.local.json
```

Comment:
Don't think we need to ignore this file.

Reply:
I add this file t ignore because I mount this file in docker-compose.local file. but as you direct point the file to .config folder, I can delete it as well

Reply:
If I understand your point correctly, I think we should get rid of it. `.local` files are a template and we need them as part of our code base.
Also, any file mounted on docker container, doesn't get affected with `.gitignore` anyways.